Applications/web_portal/web_app/library/source/pages/dashboard_page/notification_button.tsx

```diff
@@ -1,24 +1,49 @@
+import {css, StyleSheet} from 'aphrodite';
 import * as React from 'react';
 
 /** The properties used to render a NotificationButton. */
 export interface Properties {
 
-  /** The number of notification items. */
-  items: number;
+  /** The width of the button. */
+  width: number | string;
```

Comment:
Do not modify the public API of this element, which includes its Properties. Remove the width/height props and use the fixed values provided in the spec. Restore the items and isOpen prop.